openmc/cell.py

```diff
@@ -368,8 +393,19 @@ def get_cell_instance(self, path, distribcell_index):
 
         return offset
 
-    def get_all_nuclides(self):
-        """Return all nuclides contained in the cell
+    def get_nuclides(self):
+        """Returns all nuclides in the cell
+
+        Returns
+        -------
+        nuclides : list
```

Comment:
To be consistent with other parts of the API, maybe make this `list of openmc.Nuclide`.